cla_backend/libs/eligibility_calculator/cfe_civil/__init__.py

```diff
@@ -0,0 +1,33 @@
+# translate bank_balance, investment_balance and asset_balance from CLA into CFE format
+# credit_balance isn't captured (and sounds like a credit card balance) which is not part of eligibility calculations
```

Comment:
We shouldn't assume stuff like this - let's do the analysis to get a definite answer

Reply:
but this is not to do with employment anyway, so can leave it til later

Reply:
I've un-tangled the employment and savings PRs as they didn't really benefit from being together